src/Game/Maze/Maze.java

```diff
@@ -0,0 +1,93 @@
+package Game.Maze;
+import java.util.Scanner;
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+public class Maze extends JFrame  {

```

Comment:
chỉnh lại 1 tí:

- cái này dùng thằng file [`GameScreen.java`](https://github.com/team-dut/i-ran-out-of-name-help/blob/main/src/Game/Screen/GameScreen.java) luôn.
```suggestion
public class GameScreen extends Screen {
```